package.json

```diff
@@ -55,7 +55,7 @@
     "@types/linkify-it": "^3.0.1",
     "@types/lodash.escaperegexp": "^4.1.6",
     "@types/lodash.isempty": "^4.4.6",
-    "@types/node": "^15.3.0",
+    "@types/node": "^24.10.1",
```

Comment:
Why is this included here? It odesn't seem to have anything to do with the rest of the PR, and it's not aligned e.g. with our CI tests, which are currently on Node.js 22. This change should be left out.